scss/2-tools/_typography.scss

```diff
@@ -4,35 +4,15 @@
     update the list of elements that we want to check against.
 */
 @mixin cads-adjacent-heading-margin($m) {
-  h2 + &,
-  h3 + &,
-  h4 + &,
-  p + &,
-  ul + &,
-  img + &,
-  details + &,
-  table + &,
-  .cads-callout + & {
+  .cads-prose h2 + &,
+  .cads-prose h3 + &,
+  .cads-prose h4 + &,
+  .cads-prose p + &,
+  .cads-prose ul + &,
+  .cads-prose img + &,
+  .cads-prose details + &,
+  .cads-prose table + &,
+  .cads-prose .cads-callout + & {
     margin-top: $m;
   }
 }
-
-@mixin cads-paragraph() {
```

Comment:
Guessing this is because we're standardising on placeholder extends? Fine if so!